pty/ptytest/ptytest.go

```diff
@@ -72,11 +72,29 @@ func create(t *testing.T, ptty pty.PTY, name string) *PTY {
 		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitShort)
 		defer cancel()
 
-		_ = out.Close()
-		_ = ptty.Close()
```

Comment:
Of course the submitted solution is more robust, but I'm wondering if it doesn't help if you just change the order here, so:

```
ptty.Close()
out.Close()
```

Reply:
My hunch is that if we're running to an error here, it's `ptty.Close()` that hangs. Only changing the order _could_ help, but if it doesn't then we would never hit the timeout (if this is the error).

Reply:
Well, let's go for the simple approach first. I've updated the PR, what do you think?

Reply:
👍 

Ship it!